components/sst_importer/src/sst_importer.rs

```diff
@@ -883,20 +916,20 @@ mod tests {
 
         // Test ImportDir::create()
         {
-            let _file = dir.create(&meta).unwrap();
+            let _file = dir.create(&meta, None).unwrap();
             assert!(path.temp.exists());
             assert!(!path.save.exists());
             assert!(!path.clone.exists());
             // Cannot create the same file again.
-            assert!(dir.create(&meta).is_err());
+            assert!(dir.create(&meta, None).is_err());
         }
 
         // Test ImportDir::delete()
         {
             File::create(&path.temp).unwrap();
             File::create(&path.save).unwrap();
             File::create(&path.clone).unwrap();
-            dir.delete(&meta).unwrap();
+            dir.delete(&meta, None).unwrap();
```

Comment:
Please add a test to make sure delete works with TDE. Also please make sure key_manger does not contain a file that has been cleaned up.